Src/BootCamp.Chapter/TextTable.cs

```diff
@@ -38,7 +38,84 @@ public static class TextTable
         /// </summary>
         public static string Build(string message, int padding)
         {
-            return "";
+            if (message.Length == 0)
+            {
+                return ""; 
+            }
+            // make a array so I can find the longest word. 
+
+            var arrayOfWords = message.Split(Environment.NewLine);
+
+            // Find length of longest word 
+           var longestWord = ""; 
```

Comment:
Your comment and your variable name do describe different things.
Your comment is leading towards a more general solution in my opinion.

Reply:
changed